R/dp_init.R

```diff
@@ -168,6 +168,10 @@ dp_init <- function(project_path = fs::path_wd(),
     fs::dir_create(glue::glue("{project_path}/output_files"))
   }
 
+  pins_version <- utils::packageVersion("pins")
+
+  is_legacy <- pins_version >= '1.2.0'
```

Comment:
@geauxdojang this flag is backwards: we consider data products legacy if they use the old version of pins